src/server/controllers/users.controller.js

```diff
@@ -36,18 +37,49 @@ export function getAll(req, res, next) {
     });
 }
 
-// Get all user list except logged in user
 export function getMentorList(req, res, next) {
-  User.find({ _id: { $ne: req.user._id }, mentorMode: { $ne: false } })
-    .sort({ stamp_login: -1 }).exec()
-    .then(mentorList => {
-      res.status(200).json(mentorList);
+  let exceptList = [];
+  let match = { mentor_id: ObjectId(req.user._id) };
+  let project = { mentee_id: 1 };
```

Comment:
It's for projection.
We can get specific fields from aggregate result and What I want is mentee_id!
The result is like 
```
[ { _id: 5845058577fd976cb25ef0b0,
    mentee_id: 584123581282dd570c7cf093,
    list: [ [Object] ] } ]
```
Cause default field id _id.